buildspec.yml

```diff
@@ -43,18 +42,6 @@ phases:
           echo '[{
           "repository": "sagemaker-tensorflow-serving",
           "tags": [{
-            "source": "1.11.1-cpu",
```

Comment:
why are these being removed?

Reply:
Yuan and I have discussed offline that we are only releasing the latest new versions.

Reply:
Based on offline conversation, we dont expect any changes to 1.11 and 1.12 going forward, there will be no new releases of those versions either. Hence, they dont need to pass tests for any new features. Removing this will also reduce time of PR builds.

Release build has been disabled for all images